build/Build.cs

```diff
@@ -184,15 +186,68 @@ public Build()
                                 nugetVersion,
                                 FixedRuntimes.Cloud);
 
-                      DoPublish(RootProjectName, Frameworks.NetCoreApp31, nugetVersion, FixedRuntimes.Portable);
-
                       // Create the self-contained Calamari packages for each runtime ID defined in Calamari.csproj
                       foreach (var rid in Solution?.GetProject(RootProjectName).GetRuntimeIdentifiers()!)
                           DoPublish(RootProjectName, Frameworks.NetCoreApp31, nugetVersion, rid);
                   });
 
+        Target PublishCalamariFlavourProjects => _ => _
+                                                          .DependsOn(Compile)
+                                                          .Executes(() =>
+                                                                    {
+                                                                        var calamariFlavourGroup = MigratedCalamariFlavours.Flavours.Select(flavour => SourceDirectory.GlobFiles($"**/{flavour}*.csproj")); //We need Calamari & Calamari.Tests
+                                                                        foreach (var calamariFlavourProjects in calamariFlavourGroup)
+                                                                        {
+                                                                            foreach (var project in calamariFlavourProjects)
+                                                                            {
+                                                                                var calamariFlavour = XmlTasks.XmlPeekSingle(project, "Project/PropertyGroup/AssemblyName");
```

Comment:
Reading the `csproj` using the XML reader is different to what the other Targets [such as `Publish`](https://github.com/OctopusDeploy/Calamari/blob/d2ae9b171fb6119331ab02afd0793773f2a89985/build/Build.cs#L167-L192) do. It would probably be a bit cleaner and more consistent if it were possible to [use the Solution object](https://github.com/OctopusDeploy/Calamari/blob/d2ae9b171fb6119331ab02afd0793773f2a89985/build/Build.cs#L190) instead.